newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -229,86 +210,38 @@ export default function SummaryPage() {
-                  type="submit"
-                  loading={submitting}
-                  disabled={!finalDate}
-                  className={styles['finalize-button']}
-                  onClick={update}
-                >
-                  <Trans>Select final date</Trans>
-                </Button>
-              </div>
-            </>
-          )}
-        </>
-      )}
-      {isCreator && (
-        <>
-          <div className={styles['button-row']}>
-            <Button
-              size="small"
-              className={styles['delete-button']}
-              onClick={() => setDeletionModalOpen(true)}
-            >
-              <Trans>Delete this newdle</Trans>
-            </Button>
-          </div>
-          <Modal onClose={handleDeletionModalClose} size="small" closeIcon open={deletionModalOpen}>
-            <Modal.Header>
-              <Trans>Delete this newdle</Trans>
-            </Modal.Header>
-            <Modal.Content>
-              <Trans>
-                Do you really wish to delete this newdle? Please be aware that{' '}
-                <strong>this action cannot be undone.</strong>
-              </Trans>
-              {hasParticipantsWithEmail && (
-                <div className={styles['mail-checkbox']}>
-                  <Checkbox
-                    checked={sendDeletionMail}
-                    onChange={() => setSendDeletionMail(!sendDeletionMail)}
-                    label="Notify the participants via e-mail"
-                  />
-                  {sendDeletionMail && (
-                    <Input
-                      type="text"
-                      placeholder={t`Leave a comment (optional)`}
-                      className={styles['deletion-comment']}
-                      value={comment}
-                      onChange={(__, {value}) => setComment(value)}
-                    />
-                  )}
-                  {hasParticipantsWithoutEmail && (
-                    <Message className={styles['email-participant-list']}>
-                      <strong>
-                        <Trans>
-                          Some of your participants do not have e-mail addresses and will not be
-                          contacted:
-                        </Trans>
-                      </strong>
-                      <RecipientList
-                        recipients={participantsWithoutEmail}
-                        color="red"
-                        icon="close"
-                      />
-                    </Message>
-                  )}
-                </div>
-              )}
-            </Modal.Content>
-            <Modal.Actions>
+            <div className={styles['button-row']}>
               <Button
-                negative
-                onClick={() => handleDeletionModalConfirm(true)}
-                loading={deletionMailSending || deleting}
+                type="submit"
+                loading={submitting}
+                disabled={!finalDate}
```

Comment:
`!finalDate || submitting`

Reply:
Now, that I just noticed this happens in at least more than one case. What if we created a wrapper that disables the button while loading? This seems like a proper use-case, right?

Reply:
You added the new `Button` wrapper but don't actually use it in here...